<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Proxy</class>
 <widget class="QWidget" name="Proxy">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="proxylabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="copyBt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minimizeBt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimize</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startBt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
